src/auth_db/mod.rs

```diff
@@ -275,3 +275,5 @@ impl Db for DbClient {
 }
 
 unsafe impl Sync for DbClient {}
+
+unsafe impl Send for DbClient {}
```

Comment:
I think we can delete all the `Send` and `Sync` stuff from this module, now that it's not managed by Rocket. Not that it's doing any harm of course, but if we don't need it we don't need it.

Reply:
Alright :) 

Also, I just saw that the checks didn't pass. That's because there's a new version of rust and `cargo fmt -- --check` has been updated. I'll fix that as well.